linode/helper/default_value.go

```diff
@@ -13,3 +13,11 @@ func Float64PointerValueWithDefault(value *float64) basetypes.Float64Value {
 		return types.Float64Value(0)
 	}
 }
+
```

Comment:
nit: can we have a quick comment to describe the helper function?

Reply:
Fixed, thanks!